packages/editor/src/components/post-taxonomies/flat-term-selector.js

```diff
@@ -70,16 +85,107 @@ const unescapeTerms = ( terms ) => {
 	return map( terms, unescapeTerm );
 };
 
+const termNamesToIds = ( names, terms ) => {
+	return names.map(
+		( termName ) =>
+			find( terms, ( term ) => isSameTermName( term.name, termName ) ).id
+	);
+};
+
+function FlatTermSelectorMostUsed( { onSelect, taxonomy } ) {
```

Comment:
Minior: should this  be extracted to its  own file `components/post-taxonomies/most-used-terms.js` or something just to keep the size of this file contained.

Reply:
The reason didn't go with a separate file was to reuse util functions, but it might be a good idea to move them into the `utils.js` file.

Reply:
Fixed via abfa5ed.